modxcms/revolution
# Fix create button visibility in tree nodes

### What does it do?
Made css changes to hide the create button that appears in the resource tree nodes within the resource groups editing panel.

### Why is it needed?
The ability to create resources from the tree does not apply in the context of the resource groups panel.

### How to test
Clear your caches and remember to run `grunt build` within the __build/templates/default_ directory to compile the change into the main css file. Then navigate to _Content/Resource Groups_ to verify the fix.

### Related issue(s)/PR(s)
Resolves #14630.
